modules/cache/moved.tf

```diff
@@ -0,0 +1,4 @@
+moved {
```

Comment:
suggestion: I think a file name like `state_migration.tf` makes it clearer, what happens in this file. So I suggest to rename the file.